.gen/pipeline/api/openapi.yaml

```diff
@@ -19570,6 +19572,10 @@ components:
           type: string
         caFile:
           type: string
+        passwordSecretRef:
+          type: string
+        tlsSecretRef:
+          type: string
```

Comment:
Is this what we usually call secret IDs in the API?

Reply:
yes